### DISCUSSION TITLE
How to apply Syntax Coloring via TextMate?

### DISCUSSION THREAD

**Discussion by magbroxwt** [apache/netbeans#4022] [Q&A]
I tried to use TextMate as the way described in this article.[https://blogs.apache.org/netbeans/entry/lsp-client-demo-ba-sh](url)
But after running the Netbeans Module it didn't work.
![image](https://user-images.githubusercontent.com/52477885/164752943-d4bba05e-a5e5-4513-9b5b-115e23611010.png)
Whether or not I use GrammarRegistration, it always keeps the same.
By the way,I could't use clangd to replace bash-language-server as the language server which bothered me a lot.
Thanks for any help!

**Comment by negora** [comment]
@magbroxwt Have you tried to use the grammars provided here?: [Chris2011/netbeans-textmate-files](https://github.com/Chris2011/netbeans-textmate-files). They were of help to me for other languages.

**Reply by magbroxwt** [reply]
Thanks for your time. I'm glad to try it.
In client.properties I find the path of JSON file garbled. Do you know the solution? Maybe it's arch-criminal.
![image](https://user-images.githubusercontent.com/52477885/166410010-efac19a6-52da-49d6-ad40-af1371065346.png)

**Reply by negora** [reply]
I guess that has been caused by the program that you've used to minify your JSON file. If you wrote the path correctly in the original file, then that's the only reason I can think about.
